__tests__/app.test.js

```diff
@@ -330,3 +330,50 @@ describe("sort by queries, pass in two queries and order articles by them by ASC
       });
   });
 });
+
+describe("GET /api/articles?topics=topcis", () => {
+  test("should return the articles that match the certain topic", () => {
+    return request(app)
+      .get("/api/articles?topics=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.rows.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            topic: "mitch",
+            title: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            article_img_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return all articles when passed no topic", () => {
```

Comment:
We don't need this test here, as you will have already tested for it on task 5 👍